src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -26,15 +29,12 @@
 public class Kaldb {
   private static final Logger LOG = LoggerFactory.getLogger(Kaldb.class);
 
-  private final SimpleMeterRegistry meterRegistry;
+  private final PrometheusMeterRegistry prometheusMeterRegistry;
 
   public Kaldb(Path configFilePath) throws IOException {
     LOG.info("Starting Metrics setup.");
-    // TODO: Initialize metrics and set up a metrics end point.
-    // Metrics.addRegistry(Metrics.globalRegistry);
-    meterRegistry = new SimpleMeterRegistry();
-    Metrics.addRegistry(meterRegistry);
-    // TODO: Expose a metrics scraper endpoint.
+    prometheusMeterRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
```

Comment:
could this be a static final?

Reply:
`Kaldb` is initialed once so in some ways it's not needed as of now, but good so that we don't make any mistakes down the road?

Reply:
Good idea! Changed!